### ISSUE TITLE
Unable to backpropagate due to inplace operations

### ISSUE THREAD

**Issue by jeffreyjeffreywang** [FelixOpolka/STGCN-PyTorch#4]
When running main.py, a runtime error occurs: one of the variables needed for gradient computation has been modified by an inplace operation. What are the possible lines that cause this issue? Is it line 83 (torch.einsum) in stgcn.py?
